### PR TITLE
Data Science Resources Update

### PR DESCRIPTION
**Pull request by woosal1337**
## What type of PR is this? (check all applicable)

- [x] Added new link to category
- [ ] New category added
- [ ] Fixed broken link
- [ ] Typo fix
- [ ] Readme update
- [ ] Optimization
- [ ] Documentation update
- [ ] Frontend update

## Description
New Data Science Resources were added to `docs/data-science/statistical-learning.md`.

### CODE REVIEW

File: docs/data-science/statistical-learning.md [hunk 1]
```diff
@@ -5,9 +5,23 @@ sidebar_label: Statistical Learning
 ---
 
 ## Beginner
+- [Beginner Kaggle Data Science Project Walk-Through (Titanic)](https://www.youtube.com/watch?v=I3FBJdiExcg)
```
Comment (damla) [inline docs/data-science/statistical-learning.md:8]:
```suggestion
## Beginner

- [Beginner Kaggle Data Science Project Walk-Through (Titanic)](https://www.youtube.com/watch?v=I3FBJdiExcg)
```

File: docs/data-science/statistical-learning.md [hunk 2]
```diff
@@ -5,9 +5,23 @@ sidebar_label: Statistical Learning
...
 ## Intermediate
 
 - [Data Science Full Course | Simplilearn](https://www.youtube.com/watch?v=Vn7-jOUbVpw)
+- [Data Analysis with Python - Full Course for Beginners (Numpy, Pandas, Matplotlib, Seaborn)](https://www.youtube.com/watch?v=r-uOLxNrNk8)
+- [Learn Data Science Tutorial - Full Course for Beginners](https://www.youtube.com/watch?v=ua-CiDNNj30)
+- [Python for Data Science and Machine Learning Bootcamp - Udemy](https://www.udemy.com/course/python-for-data-science-and-machine-learning-bootcamp/)
 
 ## Advanced
+
+## Useful Websites
```
Comment (damla) [inline docs/data-science/statistical-learning.md:20]:
```suggestion
```

File: docs/data-science/statistical-learning.md [hunk 3]
```diff
@@ -5,9 +5,23 @@ sidebar_label: Statistical Learning
...
+- [DataSchool](https://www.dataschool.io/)
+- [100+ Free Data Science Books](https://www.learndatasci.com/free-data-science-books/)
+- [Hackr.io - Data Science](https://hackr.io/data-science)
+
+## Books
+
+## Codes
```
Comment (damla) [inline docs/data-science/statistical-learning.md:27]:
```suggestion
```

**Review by damla — COMMENTED** [review]
Hi @woosal1337,

First of all, thank you for sharing these wonderful resources 🥳

I did some changes because of the standards we have in our categories. Main titles are separated as **Beginner**, **Advanced** and **Intermediate** on purpose. Therefore, I had to delete the resources found under **Useful Websites** title you add. If you want, I can wait you to add them under another title we already have.

**Comment by woosal1337** [comment]
Sure! I will also head over to add some more. Do you mind if I mention you again here when I am completely done?

**Comment by damla** [comment]
Absolutely, feel free to mention me anytime. I will take care of it as soon as I am available 👍🏻

**Review by damla — APPROVED** [review]
Thanks for sharing 🚢🚢

**Pull request merged**